math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1222,12 +1222,16 @@ public:
 
    reference operator[](size_type idx)
    {
+#ifndef NDEBUG
       R__ASSERT(idx < size());
+#endif
```

Comment:
(only half kidding)
```suggestion
     assert(R__ASSERT(idx < size()));
```

Reply:
I like this in that it shows very clearly that `R__ASSERT` has a problem :smile: otoh I think readers will be puzzled when coming across this, I'd rather keep the more explicit ifndef